t

```diff
@@ -0,0 +1 @@
+Y2DIR=src:../yast-yast2/library/cwm/src LANG=cs_CZ.UTF-8 /sbin/yast2 partitioner_testing "$@"
```

Comment:
this should not be part of PR. Especially when you do not use yast-mass, so probably won't work for others.